CMakePresets.json

```diff
@@ -0,0 +1,57 @@
+{
+  "version": 3,
+  "cmakeMinimumRequired": {
+    "major": 3,
+    "minor": 21,
+    "patch": 0
+  },
+  "configurePresets": [
+    {
+      "name": "base",
+      "hidden": true,
+      "generator": "Ninja",
+      "binaryDir": "${sourceDir}/build/${presetName}",
+      "cacheVariables": {
+        "CCCL_ENABLE_LIBCUDACXX": false,
+        "CCCL_ENABLE_CUB": false,
+        "CCCL_ENABLE_THRUST": false,
+        "CCCL_ENABLE_TESTING": false,
+        "CCCL_ENABLE_EXAMPLES": false
+      }
+    },
+    {
+      "name": "examples",
+      "displayName": "CCCL Examples",
+      "description": "Configure, build, and run the CCCL Examples",
+      "inherits": "base",
+      "cacheVariables": {
+        "CMAKE_CUDA_ARCHITECTURES": "70",
```

Comment:
Yep -- this can be overridden by either passing `-DCMAKE_CUDA_ARCHITECTURES` to cmake (in addition to `--preset`), or modifying the config with ccmake/cmake-gui.

I only added it because the example uses libcu++ atomics, so the default arch (5.2) will throw errors for being too old. If the example avoided atomics we could drop this completely.